readthedocsext/theme/templates/account/saml.html

```diff
@@ -0,0 +1,18 @@
+{% extends "socialaccount/base_entrance.html" %}
+
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+{% block head_title %}
+  {% trans "Sign in with SAML" %}
+{% endblock head_title %}
+
+{% block content %}
+  <h1>{% trans "Continue with SAML single sign-on" %}</h1>
```

Comment:
This heading could match the other views better. I would reduce the heading size as h1 is too large and make the text match the head title.

```suggestion
  <h1 class="ui medium header">{% trans "Log in with single sign-on" %}</h1>
```